### PR TITLE
Support helm deploy in k8s

### PR DESCRIPTION
**Pull request by panzhen6668**
Support deploy in k8s #149
/cc @pixiake @ruiyaoOps  @zheng1

### CODE REVIEW

File: deploy/kubeeye.yaml [hunk 1]
```diff
@@ -387,7 +387,7 @@ spec:
         - --upstream=http://127.0.0.1:8080/
         - --logtostderr=true
         - --v=0
-        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
+        image: gcr.lank8s.cn/kubebuilder/kube-rbac-proxy:v0.8.0
```
Comment (pixiake) [inline deploy/kubeeye.yaml:390]: Is it better to use the  official repo for `kube-rbac-proxy` by default?
`gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0` or `kubesphere/kube-rbac-proxy:v0.8.0`

Comment (panzhen6668) [reply deploy/kubeeye.yaml:390]: gcr.io is limited access for CN user，we consider replacing it.

Comment (pixiake) [reply deploy/kubeeye.yaml:390]: I think we can replace it with `kubesphere/kube-rbac-proxy:v0.8.0`.

**Pull request merged**